README.md

```diff
@@ -1,2 +1,57 @@
 # hello-rails-back-end
-This exercise is going to have you create a Rails back-end api
+This exercise is going to have you create a Rails back-end api.
+
+## Built With
+
+- Ruby on Rails
+
+## Prerequisites
+
+- Ruby
+- Ruby on Rails
+- PostgreSQL
+
+## Install
+
+- Ruby
+- Ruby on Rails(as a gem)
+- Postgresql database
+
+## Getting Started
+
+To get a local copy of the app, run the following commands in your terminal:
+- `git clone https://github.com/stanleySimeon/hello-rails-back-end.git`
+- `cd /hello-rails-back-end`
+- `bundle install`
+
```

Comment:
- [OPTIONAL] As it is a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md) for this project, 

  - In the README file for the Rails API, add a link to your React app.
  - When submitting for a review - use a link to the pull request in your Rails API.
  - In the repo for the frontend part - include a PR with changes.
Please ensure you add the link to the frontend (React app) in the backend(this readme file).